Plugins/NativeXr/Source/NativeXr.cpp

```diff
@@ -134,6 +134,61 @@ namespace
         jsInputSource.Set("gripSpace", Napi::External<decltype(inputSource.GripSpace)>::New(env, &inputSource.GripSpace));
     }
 
+    void SetXRHandInputSource(Napi::Object& jsInputSource, xr::System::Session::Frame::InputSource& inputSource)
+    {
+        // Don't set hands up if hand data isn't supported/available
+        if (!inputSource.JointsTrackedThisFrame)
+        {
+            return;
+        }
+
+        constexpr const size_t HAND_JOINT_COUNT{25};
```

Comment:
nit: This pattern is fine, but another option might be to just hard-code this number into the declaration of `HAND_JOINT_NAMES`, then use `HAND_JOINT_NAMES.size()` instead of the second `constexpr` variable.